e2e/interchaintestv8/relayer_test.go

```diff
@@ -305,6 +306,128 @@ func (s *RelayerTestSuite) RecvPacketToEthTest(
 	}))
 }
 
+func (s *RelayerTestSuite) Test_2_ConcurrentRecvPacketToEth_Groth16() {
+	// I've noticed that the prover network drops the requests when sending too many
+	ctx := context.Background()
+	s.ConcurrentRecvPacketToEthTest(ctx, operator.ProofTypeGroth16, 2)
+}
+
+func (s *RelayerTestSuite) ConcurrentRecvPacketToEthTest(
+	ctx context.Context, proofType operator.SupportedProofType, numOfTransfers int,
```

Comment:
maybe we could rename this to `numConcurrentPackets` or `numConcurrentTransfers` to make it more clear (especially since we use the same name elsewhere where this will be a single packet.

Reply:
done